torch/_dynamo/exc.py

```diff
@@ -236,10 +236,13 @@ def format_error_msg_verbose(exc, code, record_filename=None, frame=None):
             code,
         )
     )
-    msg += "=" * 10 + " TorchDynamo Stack Trace " + "=" * 10 + "\n"
-    msg += format_exc()
+    line_loc = msg.find("\n")
+    header = msg[:line_loc]
```

Comment:
nit: you can do header, msg = msg.split("\n", maxsplit=1)

however, it seems like it would be even better to have format_bytecode, you know, only do bytecode, and render the header string separately. The way you've done it here seems unnecessarily contorted. If you want to avoid rendering co_name/etc multiple times... it's fine, don't worry about it, the bytecode string is so long anyway.